util/bloom_test.cc

```diff
@@ -56,6 +56,23 @@ static int NextLength(int length) {
   return length;
 }
 
+static double EstimateRate(double k, double m, double n) {
```

Comment:
Already available from the builder. See for example https://github.com/facebook/rocksdb/blob/b6d8e3674119c2d15f0d1143720871603683c502/util/filter_bench.cc#L399